docs/index.rst

```diff
@@ -85,9 +85,16 @@ Using this toolkit you can plan, execute and analyse free energy calculations us
    reference/index
    CHANGELOG
 
-Indices and tables
-------------------
+Other OpenFE Ecosystem Projects:
+--------------------------------
 
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
+**openfe** is Open Free Energy's user-facing software for performing alchemical free energy calculations.
+Below are other software projects the Open Free Energy team maintains, many of which are used by **openfe** itself.
+
+* `Lomap <https://github.com/OpenFreeEnergy/Lomap/>_`: planning perturbation networks for free energy calculations
```

Comment:
Maybe logically bunch up lomap, kartograf and konnektor? Also put the K starting projects first since we want folks to use them more.